aries_cloudagent/vc/vc_ld/manager.py

```diff
@@ -311,16 +317,17 @@ async def _get_suite_for_credential(
 
         return suite
 
-    async def _get_all_suites(self):
+    async def _get_all_proof_suites(self) -> List[LinkedDataProof]:
         """Get all supported suites for verifying presentation."""
-        suites = []
-        for suite, key_type in SIGNATURE_SUITE_KEY_TYPE_MAPPING.items():
-            suites.append(
-                suite(
+        return [
+            cast(
```

Comment:
This section could use a short comment to understand what is happening, there's a lot going on in the return statement.

I could suggest:
`# We iterate through the PROOF_KEY_TYPE_MAPPING dictionary to return an array of it's items as LinkedDataProof objects`